python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -171,6 +171,10 @@ class LoadReferenceObjectsTask(pipeBase.Task):
     ConfigClass = LoadReferenceObjectsConfig
     _DefaultName = "LoadReferenceObjects"
 
+    def __init__(self, butler=None, *args, **kwargs):
+        pipeBase.Task.__init__(self, *args, **kwargs)
+        self.butler = butler
```

Comment:
Please add a doc string and describe the butler argument. Perhaps explain briefly why it is present.